bears/python/PyDocStyleBear.py

```diff
@@ -4,7 +4,7 @@
 
 
 @linter(executable='pydocstyle',
-        use_stdout=False,
+        use_stdout=True,
         use_stderr=True,
```

Comment:
Ordinarily, I would agree with you.

However, as this is a violent change of behaviour in pydocstyle, I'd prefer that it is an explicit `use_stdout=True` here.